ooniprobe/View/TestResults/TestSummaryTableViewController.m

```diff
@@ -16,16 +18,17 @@ - (void)viewDidLoad {
     [self.navigationController.navigationBar setBackgroundImage:[UIImage new] forBarMetrics:UIBarMetricsDefault];
     self.tableView.tableFooterView = [UIView new];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(resultUpdated:) name:@"resultUpdated" object:nil];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(reloadMeasurements) name:@"uploadFinished" object:nil];
```

Comment:
I guess emitting this notification is fine, even though we've not implemented the upload of measurements yet.

Reply:
Yet, I'd remove this code and reintroduce it later.